src/Microsoft.Security.Utilities.Core/IdentifiableSecrets.cs

```diff
@@ -995,7 +1000,18 @@ internal static byte[] ConvertFromBase64String(string text)
     {
         text = TransformToStandardEncoding(text);
         text += RetrievePaddingForBase64EncodedText(text);
-        return Convert.FromBase64String(text);
+        byte[] result;
+
+        try
+        {
+            result = Convert.FromBase64String(text);
+        }
+        catch
+        {
+            return null;
```

Comment:
Please make this `bool TryConvertFromBase64String(string, out byte[])`. I'm not expecting something called ConvertXxx to return null and I wouldn't notice in PR if someone forgot to check.

Reply:
It appears there's another caller that was not updated: GetBase64EncodedSignatureBytes.